math/mcmcf90/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           compilers 1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        mjlaine mcmcf90 95fb3b4eba55ac834a7285d83063e5e5eba263b2
+version             2019.01.17
+revision            0
+categories          math science
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+license             MIT
+description         Fortran library for MCMC calculations
+long_description    This Fortran 90 library can be used to do Markov chain Monte Carlo simulation \
+                    from a posterior distribution of unknown model parameters defined \
+                    by a likelihood function and prior. The likelihood is given \
+                    as “sum-of-squares” difference of observed values from modelled values.
+homepage            https://mjlaine.github.io/mcmcf90
+checksums           rmd160  4135dd4b51e5afea3eecf3f4e5bf53c1a3d67d3a \
+                    sha256  420b4e064ea345c1b5d65a8c596febbe9f341dfd5a2abade36f5a17e179cee97 \
+                    size    56099
+
+compilers.choose    fc f90
+compilers.setup     require_fortran
+
+compiler.blacklist-append \
+                    *gcc-4.* {clang < 421}
+
+depends_lib-append  path:lib/libopenblas.dylib:OpenBLAS
+
+patchfiles          0001-Makefile-restore-PPC-support.patch
+
+post-patch {
+    reinplace "s|F90=gfortran|F90=${configure.fc}|g" ${worksrcpath}/mac.mk
+    reinplace "s|F77=gfortran|F77=${configure.fc}|g" ${worksrcpath}/mac.mk
+    reinplace "s|F90=gfortran|F90=${configure.fc}|g" ${worksrcpath}/maci.mk
+    reinplace "s|F77=gfortran|F77=${configure.fc}|g" ${worksrcpath}/maci.mk
+
+    platform darwin 10 powerpc {
+        # Rosetta hack:
+        reinplace "s|maci.mk|mac.mk|g" ${worksrcpath}/Makefile
+    }
+}
+
+use_parallel_build  no
+
+destroot {
+    copy ${worksrcpath}/libmcmcrun.a ${destroot}/${prefix}/lib/
+    xinstall -d ${destroot}${prefix}/share/${name}
+    fs-traverse f ${worksrcpath} {
+        if {[file isfile ${f}] && [file extension ${f}] == ".mod"} {
+            copy ${f} ${destroot}${prefix}/share/${name}/
+        }
+    }
+}
```

Comment:
from upstream:
"If you want to use the utilities in the library other than
mcmc_main(), your code needs to have access to the generated
*.mod files during the compilation."

so why not just leave them - I personally find all this post-destroot stuff in recent PRs confusing an unnecessary.

Reply:
What do you mean just leave? There is no target for destroot. It has to be implemented manually. I copy mod files exactly because I read instructions.

Reply:
UPD. I looked for info online, and while there is no consensus on where to put `.mod` files, a more reasonable location seems to be `/include`. Changed to it.